### PR TITLE
oracled2: Bandchain and Band Oracle script

### LINKED ISSUE #1688: Full instruction on how to run BandChain with a clean environment on OSX
Talk to swit & bun on how it should look like

### PR DESCRIPTION
**Pull request by perimeko**
Fixed #1688

### CODE REVIEW

File: chain/scripts/start_bandd.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+cd ../owasm/chaindebugtests
+for f in *; do
+    if [ -d "$f" ]; then
+        RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --package $f
+        cp ../target/wasm32-unknown-unknown/debug/$f.wasm ../../chain/pkg/owasm/res
+    fi
+done
+
+cd ../../chain
+go test ./...
```
Comment (taobun) [inline chain/scripts/start_bandd.sh]: I think these codes are not a part of a deployment.

File: chain/scripts/start_bandd.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+rm -rf ~/.band*
+dropdb my_db
+createdb my_db
+
+# initial new node
+bandd init validator --chain-id bandchain --oracle band1m5lq9u533qaya4q3nfyl6ulzqkpkhge9q8tpzs
+echo "lock nasty suffer dirt dream fine fall deal curtain plate husband sound tower mom crew crawl guard rack snake before fragile course bacon range" \
+    | bandcli keys add validator --recover --keyring-backend test  --coin-type 494
```
Comment (taobun) [inline chain/scripts/start_bandd.sh]: Remove coin-type

File: chain/scripts/start_bandd.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+echo "smile stem oven genius cave resource better lunar nasty moon company ridge brass rather supply used horn three panic put venue analyst leader comic" \
+    | bandcli keys add requester --recover --keyring-backend test  --coin-type 494
```
Comment (taobun) [inline chain/scripts/start_bandd.sh]: ditto

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/cmd/bandoracled2/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+### How to run BandChain on development mode
+1. Go to chain directory
+2. Setup your PostgresSQL user, port and database name on `start_bandd.sh`
+3. run `chmod +x scripts/start_bandd.sh` to change the access permission of start_bandd.script
+4. run `./scripts/start_bandd.sh` to start BandChain
+5. If fail, try owasm pack build then run script again.
+```
+cd ../owasm/chaintests/bitcoin_block_count/
+wasm-pack build .
+cd cd ../../../chain
```
Comment (taobun) [inline chain/cmd/bandoracled2/README.md]: cd

**Review by taobun — APPROVED** [review]

File: chain/scripts/start_bandd.sh [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+cd ../owasm/chaintests
+
+for f in *; do
+    if [ -d "$f" ]; then
+        RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release --package $f
+        cp ../target/wasm32-unknown-unknown/release/$f.wasm ../../chain/pkg/owasm/res
+    fi
+done
+cd ../../chain
+go test ./...
```
Comment (taobun) [inline chain/scripts/start_bandd.sh]: Remove this

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**